pkg/machinery/constants/constants.go

```diff
@@ -474,6 +474,25 @@ const (
 	// KubeSpanDefaultPort is the default Wireguard listening port for incoming connections.
 	KubeSpanDefaultPort = 51820
 
+	// KubeSpanDefaultRoutingTable is the default routing table for KubeSpan LAN targets.
+	//
+	// This specifies the routing table which will be used for Wireguard-available destinations.
+	KubeSpanDefaultRoutingTable = 180
+
+	// KubeSpanDefaultFirewallMark is the default firewall mark to use for Wireguard encrypted egress packets.
+	//
+	// The next consecutive mark will be used by KubeSpan to mark packets which should be forced into the Wireguard interface.
```

Comment:
This comment it now outdated:  we are defining the force port explicitly next; not using the next-consecutive necessarily.

Reply:
Ok, I will remove that